### PR TITLE
Train List Displayed - Feature1

### PR DESCRIPTION
**Pull request by Nevilston**
-  Train Name Displayed 
- Train Added and Validation Is Done Whether the train list is null or Train already present
- Train Delete from the list feature is added

### CODE REVIEW

File: src/main/webapp/index.jsp [hunk 1]
```diff
@@ -1,20 +1,13 @@
-<%@page import="java.util.Date"%>
-<%@page import="java.time.LocalDate"%>
-<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
-<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt"%>
 <html>
```
Comment (csys-fresher-batch-2021) [inline src/main/webapp/index.jsp]: Replace to <html lang="en">

File: src/main/webapp/trainListView.jsp [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+		
+		
+		
+		
+	
+		</table>
+		
+		
+	</main>
+</body>
```
Comment (csys-fresher-batch-2021) [inline src/main/webapp/trainListView.jsp:53]: Format files.

**Review by ghost — CHANGES_REQUESTED** [review]
Added comments

File: src/main/java/in/nevil/service/TrainService.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+import in.nevil.model.Train;
+
+public class TrainService {
+
+	private TrainService() {
+		throw new IllegalStateException("Utility class");
+	}
+	// Array initializing
+
+	private static List<Train> trainList = new ArrayList<>();
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/nevil/service/TrainService.java:15]: Add final

File: src/main/java/in/nevil/service/TrainService.java [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+	public static List<Train> getAllTrains() {
+		return trainList;
+	}
+
+	@SuppressWarnings("unlikely-arg-type")
+	public static boolean addTrain(String newtrainName, String newtrainNumber, int newavailableTickets) {
+		//  call validation and check productName
+		boolean istrain = false;
+		
+		if (trainList.contains("newtrainName")) {
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/nevil/service/TrainService.java:33]: trainList.contains("newtrainName")) 
to
trainList.contains(newtrainName)) {

File: src/main/java/in/nevil/servlet/DeleteTrainServlet.java [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+ * Servlet implementation class DeleteTrainServlet
+ */
+@WebServlet("/DeleteTrainServlet")
+public class DeleteTrainServlet extends HttpServlet {
+	private static final long serialVersionUID = 1L;
+	@Override
+	protected void doGet(HttpServletRequest request, HttpServletResponse response)
+			throws  IOException {
+		
+		response.getWriter().append("Served at: ").append(request.getContextPath());
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/nevil/servlet/DeleteTrainServlet.java:21]: Removed this line.

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**